nectarchain/user_scripts/fbrun/ComputeCalibCoefs.py

```diff
@@ -191,12 +197,13 @@ def findrun_firstfile(run_number):
     run_number = sys.argv[2]
     ped_run_number = sys.argv[3]
     gain_run_number = sys.argv[4]
+    ff_ttypes = int(sys.argv[5])

```

Comment:
In order to avoid misuse of the code, could it be possible to add some checks of acceptable values for `ff_types` ?